docs/tutorials/send_receive.md

```diff
@@ -0,0 +1,154 @@
+  `62626`.
+* have queues to exchange messages on. These are configured by the broker
+  properties. Two queues are setup, one called `APP.JOBS` that is of type
+  `ANYCAST` and one called `APP.COMMANDS` that is of type `MULTICAST`.
+
+```shell script
+$ kubectl apply -f - <<EOF
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: default
+  namespace: activemq-artemis-operator
+spec:
+  acceptors:
+    - bindToAllInterfaces: true
+      name: acceptAll
+      port: 62626
+  brokerProperties:
+    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
+    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
+    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
+  deploymentPlan:
+    image: 'quay.io/artemiscloud/activemq-artemis-broker-kubernetes:latest'
+    initImage: 'quay.io/artemiscloud/activemq-artemis-broker-init:latest'
+    requireLogin: false
+    size: 1
+EOF
+activemqartemis.broker.amq.io/default created
+```
+
+Wait for the Broker to start (status: `running`).
+
+```shell script
+$ kubectl get pod --namespace activemq-artemis-operator
+NAME                                                   READY   STATUS    RESTARTS
+activemq-artemis-controller-manager-66dbbbb679-hfmsn   1/1     Running   5 (3d22h ago)
+default-ss-0                                           1/1     Running   0
+```
+
+### Step 3 - Forwarding ports
+
+```shell script
+$ kubectl port-forward default-ss-0 62626 -n activemq-artemis-operator
+Forwarding from 127.0.0.1:62626 -> 62626
+Forwarding from [::1]:62626 -> 62626
+```
+
+### Step 4 - Exchanging messages between a producer and a consumer
+
+#### ANYCAST
+
+```shell script
+$ ./artemis producer --destination APP.JOBS  --url tcp://localhost:62626
+
+Connection brokerURL = tcp://localhost:62626
+Producer ActiveMQQueue[APP.JOBS], thread=0 Started to calculate elapsed time ...
+Producer ActiveMQQueue[APP.JOBS], thread=0 Produced: 1000 messages
+Producer ActiveMQQueue[APP.JOBS], thread=0 Elapsed time in second : 8 s
+Producer ActiveMQQueue[APP.JOBS], thread=0 Elapsed time in milli second : 8711 milli seconds
+```
+
+```shell script
+$ ./artemis consumer --destination APP.JOBS  --url tcp://localhost:62626
+
+Connection brokerURL = tcp://localhost:62626
+Consumer:: filter = null
+Consumer ActiveMQQueue[APP.JOBS], thread=0 wait until 1000 messages are consumed
+Received 1000
+Consumer ActiveMQQueue[APP.JOBS], thread=0 Consumed: 1000 messages
+Consumer ActiveMQQueue[APP.JOBS], thread=0 Elapsed time in second : 0 s
+Consumer ActiveMQQueue[APP.JOBS], thread=0 Elapsed time in milli second : 61 milli seconds
+Consumer ActiveMQQueue[APP.JOBS], thread=0 Consumed: 1000 messages
+Consumer ActiveMQQueue[APP.JOBS], thread=0 Consumer thread finished
+```
+
+#### MULTICAST
+
+In `n` other terminal(s) connect `n` consumer(s):
+
+```shell script
+$ ./artemis consumer --destination topic://APP.COMMANDS  --url tcp://localhost:62626
```

Comment:
Users often don't know that they need to start consumers before producer for auto-created queues. Maybe a note and/or a pointer to the Apache ActiveMQ Artemis doc could help.